core/dbt/parser/unit_tests.py

```diff
@@ -179,52 +179,51 @@ def __init__(self, schema_parser: SchemaParser, yaml: YamlBlock) -> None:
 
     def parse(self) -> ParseResult:
         for data in self.get_key_dicts():
-            unit_test_suite = self._get_unit_test_suite(data)
-            model_name_split = unit_test_suite.model.split()
-            tested_model_node = self._find_tested_model_node(unit_test_suite)
+            unit_test = self._get_unit_test(data)
+            model_name_split = unit_test.model.split()
+            tested_model_node = self._find_tested_model_node(unit_test)
 
-            for test in unit_test_suite.tests:
-                unit_test_case_unique_id = (
-                    f"unit.{self.project.project_name}.{unit_test_suite.model}.{test.name}"
-                )
-                unit_test_fqn = [self.project.project_name] + model_name_split + [test.name]
-                unit_test_config = self._build_unit_test_config(unit_test_fqn, test.config)
-
-                # Check that format and type of rows matches for each given input
-                for input in test.given:
-                    input.validate_fixture("input", test.name)
-                test.expect.validate_fixture("expected", test.name)
-
-                unit_test_definition = UnitTestDefinition(
-                    name=test.name,
-                    model=unit_test_suite.model,
-                    resource_type=NodeType.Unit,
-                    package_name=self.project.project_name,
-                    path=self.yaml.path.relative_path,
-                    original_file_path=self.yaml.path.original_file_path,
-                    unique_id=unit_test_case_unique_id,
-                    given=test.given,
-                    expect=test.expect,
-                    description=test.description,
-                    overrides=test.overrides,
-                    depends_on=DependsOn(nodes=[tested_model_node.unique_id]),
-                    fqn=unit_test_fqn,
-                    config=unit_test_config,
-                )
-                self.manifest.add_unit_test(self.yaml.file, unit_test_definition)
+            unit_test_case_unique_id = (
+                f"{NodeType.Unit}.{self.project.project_name}.{unit_test.model}.{unit_test.name}"
```

Comment:
Is there a reason we were hard coding `unit` here?  It doesn't match how we assign the unique ids for other node types.

Reply:
I don't think there's an explicit reason. What would be a more conventional way to build the unique_id here?

Reply:
Using `{NodeType.Unit}` instead of a hardcoded `unit` is more conventional.  I went ahead and made the change because it didn't seem intentional but wanted to check in case.